analysis/filter/StructureModifier/src/main/kotlin/de/maibornwolff/codecharta/filter/structuremodifier/ExtractFirstOptionPreprocessor.kt

```diff
@@ -0,0 +1,33 @@
+package de.maibornwolff.codecharta.filter.structuremodifier
+
+import mu.KotlinLogging
+import picocli.CommandLine
+import java.util.*
+
+class ExtractFirstOptionPreprocessor : CommandLine.IParameterPreprocessor {
```

Comment:
Maybe we need the extract of the first option but it should fail if more are specified.

Reply:
The documentation specifes that only one action can be performed at a time.
In my opinion, logging a warning which exact action will be performed when mutiple actions are specified is rather cumbersome (addtitional argument preprocessor) and doesn't much value.
For now a warning is logged that too many actions are specified and the command gets canceled.